src/features/search/utils/station-search.ts

```
@@ -0,0 +1,240 @@
+  let exactWordMatches = 0;
+  let partialWordMatches = 0;
+  let positionScore = 0;
+
+  // Check each query keyword against station keywords
+  for (const queryWord of queryKeywords) {
+    // Calculate exact word matches
+    if (stationKeywords.includes(queryWord)) {
+      exactWordMatches++;
+
+      // If it's the first word/prefix of the station name, give extra score
+      if (normalizedName.startsWith(queryWord + ' ')) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)